ports/chromium_bsu/Chromium BSU.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+# PORTMASTER:chromium_bsu.zip, Chromium BSU.sh
```

Comment:
This is added automatically when the release of the port is build (I have forgotten to tell you sorry)

```suggestion
```